test/distributed/test_c10d_nccl.py

```diff
@@ -3728,7 +3728,7 @@ def test_allgather_base(self):
     @parametrize("float8_dtype", [torch.float8_e4m3fn, torch.float8_e5m2])
     def test_allgather_float8(self, float8_dtype):
         device = torch.device(f"cuda:{self.rank:d}")
-        if not sm_is_or_higher_than(device, 9, 0):
+        if not sm_is_or_higher_than(device, 9, 0):  # noqa: F821
```

Comment:
I think this is a mistake - https://github.com/pytorch/pytorch/pull/153167 caused this and was since reverted...